src/routes/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -31,4 +34,9 @@ export default (safeAddress: string) => async (dispatch) => {
   if (incomingTransactions.get(safeAddress).size) {
     dispatch(addIncomingTransactions(incomingTransactions))
   }
+  const allTransactions = await loadAllTransactions(safeAddress)
```

Comment:
if we add it here, we will fetch in production `allTransactions`. That's wrong, because we have the tab off. 
We should have a separated file/call for it. in the call we should add the condition to only fetch if we are in development.

Reply:
it's fixed now, I added a `useFetchNewTransactions` hook and a `newTransactionsSelector` and used It on that tab